scripts/run_local.sh

```diff
@@ -1,47 +1,30 @@
 #!/bin/bash
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-PROJECT_ROOT=${SCRIPT_DIR}/..
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
-# sane defaults, can be overridden in my_vars
-BROKER_CMD=${PROJECT_ROOT}/broker
-GENERATED_BROKER_CONFIG=${PROJECT_ROOT}/etc/generated_local_development.yaml
+BROKER_CMD=${ASB_ROOT}/broker
 
-MY_VARS="${SCRIPT_DIR}/my_local_dev_vars"
-if [ ! -f $MY_VARS ]; then
-  echo "Please create $MY_VARS"
-  echo "cp $MY_VARS.example $MY_VARS"
-  echo "then edit as needed"
-  exit 1
-fi
-
-source ${MY_VARS}
-if [ "$?" -ne "0" ]; then
-  echo "Error reading in ${MY_VARS}"
-  exit 1
-fi
-
-if [ -z "${BROKER_CMD}" ]; then
-  echo "Please ensure BROKER_CMD is defined in ${MY_VARS}"
-  exit 1
-fi
+asb::load_vars
+asb::validate_var "BROKER_CMD" $BROKER_CMD
+asb::validate_var "OPENSHIFT_SERVER_HOST" $OPENSHIFT_SERVER_HOST
+asb::validate_var "OPENSHIFT_SERVER_PORT" $OPENSHIFT_SERVER_PORT
 
 export KUBERNETES_SERVICE_HOST=${OPENSHIFT_SERVER_HOST}
 export KUBERNETES_SERVICE_PORT=${OPENSHIFT_SERVER_PORT}
 
-BROKER_CONFIG=${GENERATED_BROKER_CONFIG}
+BROKER_CONFIG=$GENERATED_BROKER_CONFIG
 if [ ! -z "$1" ]; then
-  BROKER_CONFIG="$1"
+    BROKER_CONFIG="$1"
```

Comment:
2 or 4 spaces? I was a 4 space person, but got used to the 2 on shell scripts. I'll let the team decide, but we need to make them ALL the same. I found a tab in one file. And `prep_local_devel_env.sh` has 2 and 4.

Reply:
I say `4 > 2`.

Reply:
I prefer 2, but most importantly we should be consistent. Mixed indent is the devil.